test/Sema/enum_conformance_synthesis.swift

```diff
@@ -61,7 +61,7 @@ func customHashable() {
 enum InvalidCustomHashable {
   case A, B
 
-  var hashValue: String { return "" } // expected-error {{invalid redeclaration of synthesized implementation for protocol requirement 'hashValue'}}
+  var hashValue: String { return "" }
```

Comment:
I wonder if this is OK. We'll still complain about ambiguity at the use site for hashValue, and since it has a different type the mangling for accessors is fine. (And if the user had defined it with the right type, we wouldn't try to synthesize it or complain at all)